### PR TITLE
Refactor binpickingtask

### PR DESCRIPTION
**Pull request by felixvd**
Just reduces duplication.

### CODE REVIEW

**Comment by felixvd** [comment]
Autotester: http://autotester.test.mujin.co.jp/?scheduleid=2&pipelineid=344342

**Comment by felixvd** [comment]
I'd almost go ahead and merge this but I don't think we have a policy. @ziyan ?

File: src/binpickingtaskzmq.cpp [hunk 1]
```diff
@@ -79,23 +79,22 @@ void BinPickingTaskZmqResource::Initialize(const std::string& defaultTaskParamet
...
-void BinPickingTaskZmqResource::ExecuteCommand(const std::string& taskparameters, rapidjson::Document &pt, const double timeout /* [sec] */, const bool getresult)
-{
-    if (!_bIsInitialized) {
-        throw MujinException("BinPicking task is not initialized, please call Initialzie() first.", MEC_Failed);
-    }
-
-    if (!_zmqmujincontrollerclient) {
-        MUJIN_LOG_ERROR("zmqcontrollerclient is not initialized! initialize");
-        _zmqmujincontrollerclient.reset(new ZmqMujinControllerClient(_zmqcontext, _mujinControllerIp, _zmqPort));
+void logTaskParametersAndThrow(const std::string& taskparameters) {
```
Comment (rdiankov) [inline src/binpickingtaskzmq.cpp:82]: @felixvd function names are capitalized. i would call it: `_LogTaskParametersAndThrow`

Comment (cielavenir) [reply src/binpickingtaskzmq.cpp]: pushed

**Pull request merged**